### DISCUSSION TITLE
need good resource to learn loopback

### DISCUSSION THREAD

**Discussion by anishjain94** [loopbackio/loopback-next#7194] [Q&A]
hi, community. 
I recently started learning loopback and have been facing issues in learning loopback with proper guidance and understanding. I have tried referring to documentation but no success there as well. The mail issue that I am facing is creating any working app using documentation. Please guide me with this.

**Comment by mbnoimi** [comment]
I myself a newbie too. Follow the tutorials [here](https://loopback.io/doc/en/lb4/Tutorials.html); They are very useful.
Don't forget to [subscript to Slack](https://join.slack.com/t/loopbackio/shared_invite/zt-8lbow73r-SKAKz61Vdao~_rGf91pcsw) channel;There are many community developers available online.
**EDIT**: You can ask your questions in StackOverflow too by using [this tag](https://stackoverflow.com/questions/tagged/loopbackjs)

**Comment by achrinza** [comment]
I've heard that @\marioestradarosa's video has been a good help to many newcomers: https://youtu.be/cgBCRY169qg

For StackOverflow, please consider using the [#loopback4](https://stackoverflow.com/questions/tagged/loopback4) tag in addition to [#loopbackjs](https://stackoverflow.com/questions/tagged/loopbackjs) so that the community can distinguish between LB3 and LB4 questions. Though I would recommend asking in Slack so that we can take a more conversational approach and resolve issues faster.

**Comment by dhmlau** [comment]
There are also examples: https://loopback.io/doc/en/lb4/Examples.html that might help you.

**Comment by hacksparrow** [comment]
@anishjain94 you may find my tutorial helpful too - https://www.hacksparrow.com/loopback/tutorials/loopback-4.html.
